wx_test_paste_census.cpp

```diff
@@ -290,13 +290,15 @@ LMI_WX_TEST_CASE(paste_census)
                 gender_radiobox = dynamic_cast<wxRadioBox*>(gender_window);
             LMI_ASSERT(gender_radiobox);
 
-            wxUIActionSimulator ui;
+            UIActionSimulator ui;
             // Select the last, "Unisex", radio button, by simulating
             // down-arrow twice: female --> male, then male --> unisex.
             ui.Char(WXK_DOWN);
             wxYield();
             ui.Char(WXK_DOWN);
             wxYield();
+            ui.Char(WXK_SPACE); // And make it current (required under GTK).
```

Comment:
```suggestion
            ui.Char(WXK_SPACE); // And make it current (required under GTK and does no harm under MSW).
```
Although OTOH it's still a bit wasteful to emulate an unnecessary key press under MSW... maybe we should have some `wxUIActionSimulator::SelectRadioButton()` helper in the future.